rplugin/python3/isort_nvim.py

```diff
@@ -31,7 +32,7 @@ def __init__(self, nvim):
     def isort_command(self, args, range):
         buffer = self.nvim.current.buffer
         text = self._get_lines(buffer, range)
-        output = self._isort(text, *args)
+        output = self._isort(text, *args, cwd=os.path.dirname(buffer.name))
```

Comment:
Can we use pathlib here?

```suggestion
        output = self._isort(text, *args, cwd=Path(buffer.name).parent)
```